src/lib/run_environment.ml

```diff
@@ -94,10 +94,37 @@ module Tool = struct
   end
 end
 
+module Make_fun = struct
+  module Requirement = struct
+    type t = [
+      | `Processors of int
+      | `Internet_access
+      | `Memory of [ `GB of float | `Decent | `Big ]
```

Comment:
I also don't know what \`Decent or \`Big mean; perhaps we can leave it at \`Gb for now? And if gets too cumbersome, we can reconsider?